src/Aspire.Hosting.Azure.AppContainers/AzureContainerAppsInfrastructure.cs

```diff
@@ -608,6 +608,7 @@ BicepValue<string> GetHostValue(string? prefix = null, string? suffix = null)
                     EndpointProperty.Url => GetHostValue($"{scheme}://", suffix: isHttpIngress ? null : $":{port}"),
                     EndpointProperty.Host or EndpointProperty.IPV4Host => GetHostValue(),
                     EndpointProperty.Port => port.ToString(CultureInfo.InvariantCulture),
+                    EndpointProperty.HostAndPort=> GetHostValue(suffix: isHttpIngress ? null : $":{port}"),
```

Comment:
FYI - @davidfowl - I found a bug that we missed here and fixed it so all the generate-manifests work again.

Reply:
I didn't know if I should switch off `isHttpIngress` or not. If you explicitly ask for `HostAndPort` and this is the httpIngress, should we still tack on `:80`?

Reply:
Rethinking this, I think it should always put the port. The reasoning the URL can drop the port is because it contains the scheme at the begining. `http://` indicates what the port is. If you say `HostAndPort` we should always give you the host and the port.

Reply:
I agree, it should include host an port